natgeo/mortar
# [AEM-6648] Video Duration Not Displaying

https://jira.natgeo.com/browse/AEM-6648

To Test:
Pull AEM-6648
Verify duration shows both with and without the presence of a kicker

Also, I refactored the way the cards handle the z-index functionality of the kicker based on whether it is a link or not.
